### DISCUSSION TITLE
Umep execution error

### DISCUSSION THREAD

**Discussion by Fer374** [UMEP-dev/UMEP#633] [General]
I am having problems running the Solar Energy - Introduction to SEBE exercise in the Irradiance on building envelopes step during the visualization when running it I get the following error: 2024-07-12T22:15:04 WARNING Traceback (most recent call last):
              File “C:\Users/ferna/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\UMEP\SEBEVisual\tools\areaTool.py”, line 107, in canvasMoveEvent.
              self.areaRubberband1 = QgsRubberBand(self.canvas, False)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
             TypeError: QgsRubberBand(): argument 2 has unexpected type 'bool'.

Additionally when I run the step Irradiance map with values less than 900 kWh filtered out it does not recognize my file folder and throws this error. 
“C:users/ferna/Documents/Suews/Data input/Gotenborg_SWEREF99_1200/Goteborg_SWEREF99_1200/Goteborg_SWEREF99_1200/building_shp it is worth noting that when checking on the computer the layer is found and can be added as a layer in the main qgis interface.

On the other hand during the execution of the exercise Urban Energy Balance - SUEWS Advanced in the test phase when placing the default parameters for the exercise and call the existing folder on the computer generates the following error: Traceback (most recent call last):
File “C:\Users/ferna/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\processing_umep\processor\suews_algorithm.py”, line 206, in processAlgorithm.
if (underscorePos[1] - underscorePos[0]) == 1:
~~~~~^^^
IndexError: list index out of range

Execution failed after 0.04 seconds 

This for version 3.34 LTR QGIS version Phyton 3.12 however when changing to version 3.28 LTR QGIS version Phyton 3.9.5. the errors persist both in the first exercise and in the second one that I mentioned for this last one generates this error: 
C:\USersferna\AppData\Roaming\Python\Python39\site-packages\supy
C:\UsersersfernaAppDataRoamingPython39site-packages
Creating RunControl.nml
Traceback (most recent call last):
File “C:\Users/ferna/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\processing_umep\processor\suews_algorithm.py”, line 206, in processAlgorithm.
if (underscorePos[1] - underscorePos[0]) == 1:
IndexError: list index out of range

I remain attentive
thank you very much

**Comment by biglimp** [comment]
I think we fixed this issue but we probably did not updated the QGIS repo version of UMEP yet. Use the development version (see our webpage in how to use this).

**Reply by Fer374** [reply]
Good day install the UMEP development version however the problem persists and does not run

**Reply by biglimp** [reply]
Is it the issue on Solar Radiation tutorial or SUEWS Advanced? The fix I mentioned earlier where on SEBE Visual.
